scripts/md-gen/hook-commands/main.go

```diff
@@ -4,18 +4,14 @@
 package main
```

Comment:
Nice, this is how we should be generating the CLI docs as well.

Reply:
It's certainly an improvement, but even this isn't where we should be imo. We still rely on the `juuj/cmd` package and it's `documentation` command. Really we should bypass that whole thing

Reply:
Good point, I didn't spot that it just called that, at least this should be tied to the current version.